src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -257,6 +257,7 @@ protected override void ElapseEditorEntryTime()
     {
         // TODO: select which units will be used for the master elapsed time counter
         CurrentGame.GameWorld.OnTimePassed(1);
+        cellEditorTab.UpdateBackgroundImage(CurrentPatch.BiomeTemplate);
```

Comment:
I think a comment here would be helpful about the events being able to change the properties of the current patch.